From c6eb3899fa7b225a63c64236cf2a6c4f8db0d1ee Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:06:55 -0400
Subject: [tools] perf/ui/tui: Rename hist_browser__update_nr_entries()

Message-id: <1410181752-25631-171-git-send-email-jolsa@redhat.com>
Patchwork-id: 91806
O-Subject: [PATCH RHEL7.1 BZ1134356 170/307] perf ui/tui: Rename hist_browser__update_nr_entries()
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 112f761fc0b43def377af889f8cd242df6af9e34
Author: Namhyung Kim <namhyung@kernel.org>
Date: Tue Apr 22 14:05:35 2014 +0900

description
===========
Rename ->nr_pcnt_entries and hist_browser__update_pcnt_entries() to
->nr_non_filtered_entries and hist_browser__update_nr_entries() since
it's now used for filtering as well.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/ui/browsers/hists.c b/tools/perf/ui/browsers/hists.c
index 311226e..769295b 100644
--- a/tools/perf/ui/browsers/hists.c
+++ b/tools/perf/ui/browsers/hists.c
@@ -26,13 +26,14 @@ struct hist_browser {
  int       print_seq;
  bool       show_dso;
  float       min_pcnt;
- u64       nr_pcnt_entries;
+ u64       nr_non_filtered_entries;
 };
 
 extern void hist_browser__init_hpp(void);
 
 static int hists__browser_title(struct hists *hists, char *bf, size_t size,
     const char *ev_name);
+static void hist_browser__update_nr_entries(struct hist_browser *hb);
 
 static void hist_browser__refresh_dimensions(struct hist_browser *browser)
 {
@@ -310,8 +311,6 @@ static void ui_browser__warn_lost_events(struct ui_browser *browser)
   "Or reduce the sampling frequency.");
 }
 
-static void hist_browser__update_pcnt_entries(struct hist_browser *hb);
-
 static int hist_browser__run(struct hist_browser *browser, const char *ev_name,
         struct hist_browser_timer *hbt)
 {
@@ -322,7 +321,7 @@ static int hist_browser__run(struct hist_browser *browser, const char *ev_name,
  browser->b.entries = &browser->hists->entries;
  browser->b.nr_entries = browser->hists->nr_entries;
  if (browser->min_pcnt)
-  browser->b.nr_entries = browser->nr_pcnt_entries;
+  browser->b.nr_entries = browser->nr_non_filtered_entries;
 
  hist_browser__refresh_dimensions(browser);
  hists__browser_title(browser->hists, title, sizeof(title), ev_name);
@@ -340,8 +339,8 @@ static int hist_browser__run(struct hist_browser *browser, const char *ev_name,
    hbt->timer(hbt->arg);
 
    if (browser->min_pcnt) {
-    hist_browser__update_pcnt_entries(browser);
-    nr_entries = browser->nr_pcnt_entries;
+    hist_browser__update_nr_entries(browser);
+    nr_entries = browser->nr_non_filtered_entries;
    } else {
     nr_entries = browser->hists->nr_entries;
    }
@@ -1343,7 +1342,7 @@ close_file_and_continue:
  return ret;
 }
 
-static void hist_browser__update_pcnt_entries(struct hist_browser *hb)
+static void hist_browser__update_nr_entries(struct hist_browser *hb)
 {
  u64 nr_entries = 0;
  struct rb_node *nd = rb_first(&hb->hists->entries);
@@ -1354,7 +1353,7 @@ static void hist_browser__update_pcnt_entries(struct hist_browser *hb)
   nd = rb_next(nd);
  }
 
- hb->nr_pcnt_entries = nr_entries;
+ hb->nr_non_filtered_entries = nr_entries;
 }
 
 static int perf_evsel__hists_browse(struct perf_evsel *evsel, int nr_events,
@@ -1411,7 +1410,7 @@ static int perf_evsel__hists_browse(struct perf_evsel *evsel, int nr_events,
 
  if (min_pcnt) {
   browser->min_pcnt = min_pcnt;
-  hist_browser__update_pcnt_entries(browser);
+  hist_browser__update_nr_entries(browser);
  }
 
  fstack = pstack__new(2);
-- 
1.7.1